pants.toml

```diff
@@ -394,6 +394,7 @@ pytest_plugins.add = [
   #  understands our __init__ files. NB: this version matches 3rdparty/python/requirements.txt.
   "setuptools==44.0.0",
   "ipdb",
+  "pytest-icdiff",
```

Comment:
Changing this here will have the effect of changing it only for the `pantsbuild/pants` repo... was that your intent, or did you mean to change it by default for consumers of pants?

If the latter thing, you'd want to change it here: https://github.com/pantsbuild/pants/blob/ac5465ad5aee81ac2c1a5462148ba284fb277526/src/python/pants/backend/python/subsystems/pytest.py#L29-L40

Reply:
I don't think that we should change this for everyone by default. That's surprising. Instead, I plan to add a tip to the docs at https://pants.readme.io/docs/python-test-goal, like we already have a tip for installing `ipdb`.

Reply:
👍